src/templates/Base/styles.ts

```diff
@@ -0,0 +1,57 @@
+import styled, { css } from 'styled-components'
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100vh;
+`
+
+export const SectionHeader = styled.section`
```

Comment:
Se o nome já é `Header`, pq não usar `header`? xD

```suggestion
export const SectionHeader = styled.header`
```